ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/saas/search/_cq_editConfig.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
- [ ] What's the purpose of this empty edit config?

Reply:
Not needed. I'll remove it